x-pack/plugins/endpoint/common/types.ts

```diff
@@ -109,26 +162,78 @@ export type AlertEvent = Immutable<{
   event: {
     id: string;
     action: string;
+    category: string;
+    kind: string;
+    dataset: string;
+    module: string;
+    type: string;
   };
-  file_classification: {
-    malware_classification: {
-      score: number;
+  process: {
+    code_signature: {
+      subject_name: string;
+      trusted: boolean;
     };
-  };
-  process?: {
-    unique_pid: number;
+    command_line: string;
+    domain: string;
     pid: number;
+    ppid: number;
+    entity_id: string;
+    parent: {
+      pid: number;
+      entity_id: string;
+    };
+    name: string;
+    hash: HashFields;
+    pe: {
+      imphash: string;
+    };
+    executable: string;
+    sid: string;
+    start: number;
+    malware_classifier: MalwareClassifierFields;
+    token: {
+      domain: string;
+      type: string;
+      user: string;
+      sid: string;
+      integrity_level: number;
+      integrity_level_name: string;
+      privileges: PrivilegesFields[];
+    };
+    thread: ThreadFields[];
+    uptime: number;
+    user: string;
   };
-  host: {
-    hostname: string;
-    ip: string;
-    os: {
-      name: string;
+  file: {
+    owner: string;
+    name: string;
+    path: string;
+    accessed: number;
+    mtime: number;
+    created: number;
+    size: number;
+    hash: HashFields;
+    pe: {
+      imphash: string;
+    };
+    code_signature: {
+      trusted: boolean;
+      subject_name: string;
     };
+    malware_classifier: {
+      features: {
+        data: {
+          buffer: string;
+          decompressed_size: number;
+          encoding: string;
```

Comment:
These fields are not going to be used by the UI, right? Anything the UI won't use should be optional or even removed from the types.

Reply:
but we can do that in subsequent PRs

Reply:
adding this to this ticket elastic/endpoint-app-team#189